src/runtime_src/core/tools/xbtracer/src/lib/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+link_directories($ENV{XILINX_XRT}/lib)
+
+set(SRCS
+  capture.cpp
+  logger.cpp
+  xrt_device_inst.cpp
+)
+
+if (WIN32)
+  add_definitions( -DXCL_DRIVER_DLL_EXPORT )
+  add_definitions( -DXRT_API_SOURCE )
+endif()
```

Comment:
These should never be defined in make files.  You are setting these defines only if your compiled object code is in the same library that exports API sources, e.g. in xrt_coreutil.  See any file in core/common/api/*.cpp and how these define these ones.

Reply:
Done. Pushed in latest updates